reader-sdk-react-native-quickstart/android/app/build.gradle

```diff
@@ -220,7 +220,10 @@ dependencies {
     }
 // The version of react-native is set by the React Native Gradle Plugin
     implementation("com.facebook.react:react-android")
-    // implementation 'com.reactnative:react-native-square-reader-sdk:1.7.2'
+ //   implementation 'com.reactnative:react-native-square-reader-sdk:1.7.4'
```

Comment:
(1) BLOCKING: Why is this comment talking about 1.7.4 if we are creating 1.7.7 here?
(2) Why does this comment even exist?  It seems like the only thing it _can_ do is to be skewed from truth?

- [ ] holding approval until I know why the string 1.7.4 should be in any android/... file if the new versions, plugin and SDK both, is called 1.7.7?